tests/e2e/e2e.sh

```diff
@@ -18,12 +18,29 @@ set -eux
 
 export ARTIFACTS="${ARTIFACTS:-$(mktemp -d)}"
 ROOT=$(cd ../../../; pwd)
+HUB=istio-testing
+TAG=istio-testing
 
 function setup_docker() {
   HUB=istio-testing TAG=1.5-dev make controller docker
   kind --loglevel debug --name istio-testing load docker-image istio-testing/operator:1.5-dev
   }
 
+function build_kind_images() {
+  # Build just the images needed for the tests
+  for image in pilot proxyv2 app test_policybackend mixer citadel galley sidecar_injector kubectl node-agent-k8s; do
+     make docker.${image}
+  done
+
+  time load_kind_images
+}
+
+function load_kind_images() {
+	# Archived local images and load it into KinD's docker daemon
+	# Kubernetes in KinD can only access local images from its docker daemon.
+	docker images "${HUB}/*:${TAG}" --format '{{.Repository}}:{{.Tag}}' | xargs -n1 kind --loglevel debug --name istio-testing load docker-image
```

Comment:
I believe this may not be necessary as it was not necessary prior for e2e testing to work.  Perhaps this was a side-effect of running the `make e2e` test target in the istio repository - but there should be no strong rationale to duplicate this functionality between repositories (it will make maintenance harder).

Reply:
removed already